docs/HowToGuides/GettingStarted.md

```diff
@@ -305,16 +308,25 @@ It is even possible to integrate the Ninja build into Xcode. For details on how
   In many situations, there are several errors, so scrolling further back
   and looking at the first error may be more helpful than simply looking
   at the last error.
-- Check if others have encountered the same issue on the Swift forums or on
-  [Swift repository 'Issues' tab][Swift Issues].
-- Create a new Swift forums thread in the Development category. Include
-  information about your configuration and the errors you are seeing.
-  - You can [create a gist](https://gist.github.com) with the entire build
-    output and link it, while highlighting the most important part of the
-    build log in the post.
-  - Include the output of `utils/update-checkout --dump-hashes`.
+- Check if there is already a [Swift Forums] thread from someone else with the same issue.
+  - Here is a list with some threads that describe common issues:
+    * [Problems with `build-script` building compiler with `–xcode`](https://forums.swift.org/t/problems-with-build-script-building-compiler-with-xcode/53477)
+    * [Error building the compiler (even with ninja)](https://forums.swift.org/t/error-building-the-compiler-even-with-ninja/54834)
+    * [Build failure on Apple MacBook Pro with Apple M1 Chip](https://forums.swift.org/t/build-failure-on-apple-silicon-m1-mac-mini/45011)
+    * [CMake cannot compile a test program](https://forums.swift.org/t/build-failure-locally/55695)
+    * [Building Swift compiler from source fails when not using Ninja](https://forums.swift.org/t/building-swift-compiler-from-source-fails-when-not-using-ninja/54656)
+    * [ALL_BUILD Target failing at validation](https://forums.swift.org/t/help-building-swift-in-xcode-error/49728)
+    * [“gtest/gtest.h” not found while compiling the compiler](https://forums.swift.org/t/gtest-gtest-h-not-found-in-typeref-cpp-while-compiling-the-compiler/44399)
+  - If after going through the forums you still could not found a solution for the issue you are seeing, feel free to create a new one.
```

Comment:
The top-level bullet list ought to be interpreted as a sequence of actions. Do we really need to nest this one and introduce another level?